exchange/exchange-ps/exchange/Get-AuthenticationPolicy.md

```diff
@@ -81,10 +81,13 @@ The AllowLegacyExchangeTokens switch specifies whether legacy Exchange tokens ar
 
 Legacy Exchange tokens include Exchange user identity and callback tokens.
 
+This switch also specifies the most recent date and time that an add-in was allowed or blocked from acquiring a token.
```

Comment:
> most recent date and time

this is not exactly accurate and going to be confusing... the date updates once a week or so. In other words, it has a precision of about a week.

Reply:
Updated.